executor/partition_table_test.go

```diff
@@ -110,6 +110,80 @@ func (s *partitionTableSuite) TestPartitionUnionScanIndexJoin(c *C) {
 	tk.MustExec("commit")
 }
 
+func (s *partitionTableSuite) TestPointGetwithRangeAndListPartitionTable(c *C) {
+	if israce.RaceEnabled {
+		c.Skip("exhaustive types test, skip race test")
+	}
+	tk := testkit.NewTestKitWithInit(c, s.store)
+	tk.MustExec("create database test_pointget_list_hash")
+	tk.MustExec("use test_pointget_list_hash")
+	tk.MustExec("set @@tidb_partition_prune_mode = 'dynamic'")
+	tk.MustExec("set @@session.tidb_enable_list_partition = ON")
+
+	// list partition table
+	tk.MustExec(`create table tlist(a int, b int, unique index idx_a(a), index idx_b(b)) partition by list(a)( 
```

Comment:
Please add more cases about multiple unique index, for example:
```
mysql> create table t (a int, b int, unique key(a), unique key(b));
Query OK, 0 rows affected (0.01 sec)

mysql> explain select * from t where a=1 and b=2;
+-------------------+---------+------+---------------------+-----------------+
| id                | estRows | task | access object       | operator info   |
+-------------------+---------+------+---------------------+-----------------+
| Selection_6       | 0.00    | root |                     | eq(test.t.b, 2) |
| └─Point_Get_5     | 1.00    | root | table:t, index:a(a) |                 |
+-------------------+---------+------+---------------------+-----------------+
2 rows in set (0.00 sec)
```
To test if the extra condition will be converted to a `Selection`.

Reply:
For partition table, all the unique keys must be included in the partition key. However, now we don't support `partition by range columns(...)` or `partition by list columns(...)`. That means we only support partition table with one unique key to use point get.